lib/checks/aria/aria-valid-attr-value-evaluate.js

```diff
@@ -87,25 +85,36 @@ function ariaValidAttrValueEvaluate(node, options, virtualNode) {
     }
   };
 
-  for (let i = 0, l = attrs.length; i < l; i++) {
-    const attrName = attrs[i];
+  virtualNode.attrNames.forEach(attrName => {
+    // skip any attributes handled elsewhere (we don't want use
+    // Array.filter first as that would mean looping over the
+    // attributes twice. on a page with tens of thousands of
+    // nodes that's a lot of overhead cost in performance)
```

Comment:
This really isn't necessary.

```suggestion
```

Reply:
I felt it was since I was refactoring the code to use a filter then a foreach but realized it would cause a performance hit. Didn't want someone to make that same mistake in the future